### DISCUSSION TITLE
Odd example sentence for 縁は異なもの

### DISCUSSION THREAD

**Discussion by Morgawr** [Jitendex/Jitendex#26] [Errors and Corrections]
JP: 縁は異なもの。 (yes, the expression itself is the "full" example sentence)

EN: Marriages are made in heaven.

---

I'm not a fan of translating one proverb with another as an example usage since it's not a literal translation, but also this example sentence really says nothing that isn't already in the definition of the expression itself (which is given as: "inscrutable are the ways people are brought together" and "there is no telling how people are brought together").

I think this example sentence is rather useless and should be removed, if we don't have another example of how it can be used then it's okay to not have an example sentence at all, and maybe "Marriages are made in heaven" can be added to the word definition if we really wanted to.

**Comment by stephenmk** [comment]
There are times when it makes sense to define a Japanese proverb or idiomatic expression with an English one. I think "strike while the iron is hot" is a good fit for "善は急げ," for example.  When the literal interpretation of an idiom isn't especially clear, we usually add an extra "literal" gloss to the end of the entry's glossary and tag it as such.

Any time you see an example sentence that is exactly the same as the headword or as an already cross-referenced entry, it's safe to consider it a mistake that needs to be fixed. Now that I think about it, it probably wouldn't be too hard for me to write a program to find all of these sorts of example sentences. At any rate, thanks for reporting this one.
